client/app/services/language.service.js

```diff
@@ -100,6 +100,8 @@
 
       var code = getLocale(data);
       setLocale(code);
+      
+      return data;
```

Comment:
`Language.onReload` did not return the original data, so could not be chained [here](https://github.com/ManageIQ/manageiq-ui-self_service/pull/127/files#diff-b39a4f7482697f7f1a49c46b01c64d1fR92)